ml/NN/config/NN.conf.example

```diff
@@ -0,0 +1,17 @@
+[NN]
+mode = train
+input_file= 8_12_VT_peinfo_objdump_join.parquet
+model = NN_model
+output = predictions
+batch_size = 10
+epochs = 1
+cluster_size = 3
+steps= 1000
+
+# If you want to predict the label of the labeled data.
```

Comment:
I don't understand these lines. The comment is the same for the key but the associated values are different. 
do you mean train or predict? If so maybe have  boolean key that says 
train = True
 
train = True